walkthroughs/week-1-web-development/examples/topic_page/src/main/webapp/style.css

```diff
@@ -0,0 +1,30 @@
+.rectangle {
+  background-color:whitesmoke;
+  width: 1000px;
+  border: 10000px rgb(192, 255, 241);
+  padding: 50px;
+  margin: 10px;
+}
+
+.center {
+  text-align: center;
+  color: white;
+  font-size: 300%;
+}
+
+.white {
+    color: white;
+    font-size: 200%
+}
```

Comment:
same thing here, the styles for the topics should be in a topic-styles.css foler inside of /topics/ so that it doesn't overwrite the main styles.css file